client/components/LearnerViewDirective.js

```diff
@@ -80,6 +80,12 @@ tie.directive('learnerView', [function() {
                   <select class="tie-lang-select-menu" name="lang-select-menu">
                     <option value="Python" selected>Python</option>
                   </select>
+                  <select class="tie-question-set-select" name="question-set-select"
+                          ng-change="changeQuestionSet()" ng-model="currentQuestionSetId"
+                          ng-options="i.questionSetId as i.questionSetId for i in questionSetIds">
+                    <option style="display: none" value="">Question Set</option>
```

Comment:
Yeah -- that's done for consistency with the "Theme" dropdown, below. It also has the <option></option>, would you like me to remove that one as well?